sdk/eventhub/azure-eventhub/azure/eventhub/aio/_consumer_async.py

```diff
@@ -129,7 +134,12 @@ def _create_handler(self, auth: Union["uamqp_JWTTokenAsync", JWTTokenAuthAsync])
             self._offset,
             event_position_selector(self._offset, self._offset_inclusive),
         )
-        desired_capabilities = [RECEIVER_RUNTIME_METRIC_SYMBOL] if self._track_last_enqueued_event_properties else None
+        desired_capabilities = (
+            [RECEIVER_RUNTIME_METRIC_SYMBOL,
+            GEOREPLICATION_SYMBOL]
+            if self._track_last_enqueued_event_properties
+            else [GEOREPLICATION_SYMBOL]
```

Comment:
since this will always be passed in, if geo replication is not enabled,  its the equivalent of passing in `None`?

Reply:
If it's not enabled, I believe the service would send back an int offset instead of str.